packages/machine-extractor/src/toMachineConfig.ts

```diff
@@ -234,6 +226,13 @@ export const getActionConfig = (
           }),
         });
         return;
+      case isAssignAction(action):
+        actions.push({
+          type: action.name || `Assignment ${Math.random().toFixed(3)}`,
```

Comment:
We shouldn't auto-generate things here on behalf of the user. This kind of a thing is "lossy" because by the time we end up consuming this, we won't be able to tell apart the provided and the auto-generated strings.